embedded/sql/stmt.go

```diff
@@ -3999,3 +4006,59 @@ func (stmt *FnDataSourceStmt) resolveListIndexes(ctx context.Context, tx *SQLTx,
 
 	return newValuesRowReader(tx, params, cols, stmt.Alias(), values)
 }
+
+func NewDeleteTableStmt(table string) *DeleteTableStmt {
+	return &DeleteTableStmt{table: table}
+}
+
+// DeleteTableStmt represents a statement to delete a table.
+type DeleteTableStmt struct {
+	table string
```

Comment:
`ifExists bool`

By default it may be better to return an error if the table does not exist

Reply:
ack, done